docs/guides/theme_architecture/blocks/schema/variables/variable_types/page.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: Page
+parent: Variable types
+---
+
+# Page
+
+Renders a select input from which a Creator can select a page from their site pages.
+Returns a [Page]({% link docs/reference/objects/page.md %}) object.
+The page variable does not accept a default.
```

Comment:
Learn something new everyday! Reading below, blog accepts random, I assume not true of page?

Reply:
I've just tested this and it actually does accept random, will update